server/tracedb/sumologic.go

```diff
@@ -0,0 +1,296 @@
+package tracedb
```

Comment:
Question: don't we need an implementation like this for the Agent? We need to implement something on https://github.com/kubeshop/tracetest/tree/main/agent/workers/datastores, no?

Reply:
Also, it would be good to have a unit test for SumoLogic on the RM's Provisioner, here: https://github.com/kubeshop/tracetest/blob/main/server/provisioning/provisioning_test.go#L350